vrs/vrs-annotation-batch-wrapper.py

```diff
@@ -180,14 +217,25 @@ def main(args):
             )
         )
     else:
-        logger.info("Constructing a test table")
+        logger.info("Constructing a test Table")
         ht_final = ht_original.annotate(
             VRS_Allele=ht_annotated[ht_original.locus, ht_original.alleles].VRS_Allele
         )
 
-    logger.info(f"Outputting final table at: {output_paths_dict[args.version]}")
-    ht_final.write(output_paths_dict[args.version],overwrite=args.overwrite)
-    # hl.write_table(ht_final,output_paths_dict[args.version])
+        logger.info(f"Outputting final table at: {output_paths_dict[args.version]}")
+        ht_final.write(output_paths_dict[args.version], overwrite=args.overwrite)
+
+    # Deleting temporary files saves a great deal of space and keeps CloudFuse costs down
+    logger.info("Preparing to delete temporary files and sharded VCFs generated")
+    delete_temps = hb.Batch(name="delete_temps", backend=backend)
+    d1 = init_job_with_gcloud(
+        batch=delete_temps,
+        name=f"del_files",
+        image=args.image,
+        mount=working_bucket,
```

Comment:
does this (https://the-tgg.slack.com/archives/C048GFPPW0L/p1680550631901889), include the cost of mounting the bucket again here?

Reply:
Yep! That $50 is from the cost of using CloudFuse and all of the GCP read/write statements